packages/gatsby-theme-store/src/components/ProductPage/useAsyncProduct.ts

```diff
@@ -33,10 +33,10 @@ export const useAsyncProduct = (
 }
 
 export const query = gql`
-  query AsyncProductQuery($slug: String) {
+  query AsyncProductQuery($slug: String, $regionId: String) {
     vtex {
-      product(slug: $slug) {
-        id: productId
+      product(slug: $slug, regionId: $regionId) {
+        productId
```

Comment:
Revert back to how it was
```suggestion
        id: productId
```

Reply:
good catch, took a while to fix because i needed to re-generate the generated query file